tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = rf3, rf4
 skip_missing_interpreters = true
 [pytest]
-addopts = --benchmark-skip
+addopts = --benchmark-skip --lf
```

Comment:
I believe it's fine to leave this option. Those option are also read automatically by pytest if you just run `pytest tests` in cli. `--lf` option run last failed tests -> so if you have 4 failing tests out of 400, it will run only 4. If everything passed last run, it will run all tests.